.env.example

```
@@ -60,6 +62,15 @@ AGENT_HARNESS_SOURCE_SYNC_MAX_PAGES_PER_RUN=10
 AGENT_HARNESS_NPM_SEARCH_RESULT_LIMIT=12
 AGENT_HARNESS_NPM_MCP_SEARCH_QUERY_LIMIT=8
 
+# Discovery demand and adjacent-tooling controls.
+AGENT_HARNESS_DISCOVERY_ADJACENT_TOOLING_ENABLED=true
+AGENT_HARNESS_DISCOVERY_REGISTRY_SEARCH_MAX_TERMS=10
+AGENT_HARNESS_DISCOVERY_REGISTRY_SEARCH_MAX_RESULTS_PER_TERM=50
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENT_HARNESS_DISCOVERY_REGISTRY_SEARCH_MAX_RESULTS_PER_TERM key should go before the AGENT_HARNESS_DISCOVERY_REGISTRY_SEARCH_MAX_TERMS key

(UnorderedKey)